engine/render/src/render/font_renderer_private.h

```diff
@@ -82,13 +83,15 @@ namespace dmRender
         }
 
         void*                   m_UserData; // The font map resources (see res_font.cpp)
+        dmGraphics::HContext    m_GraphicsContext; // Used to recreate textures
```

Comment:
If you don't want to store the context, we could "expose" the GetInstalledContext() function from graphics_private.h into graphics.h. It's a bit of a hack that we needed for the glfw wrapper functions however